cwms_radar_api/src/main/java/cwms/radar/data/dao/TimeSeriesIdentifierDescriptorDao.java

```diff
@@ -58,13 +58,18 @@ public TimeSeriesIdentifierDescriptorDao(DSLContext dsl) {
     public void create(TimeSeriesIdentifierDescriptor tsid, boolean versionedFlag,
                        Number intervalForward, Number intervalBackward, boolean failIfExists
     ) {
-        BigDecimal tsCode = CWMS_TS_PACKAGE.call_CREATE_TS_CODE(dsl.configuration(),
+        dsl.connection(c -> {
+            BigDecimal tsCode = CWMS_TS_PACKAGE.call_CREATE_TS_CODE(
+                getDslContext(c,tsid.getOfficeId()).configuration(),
```

Comment:
looks like most (all?) `getDslContext` calls wind up just calling `.configuration()` could replace by just returning the `Configuration`

Reply:
That's a good point, mostly I was just trying to keep that original signature but it doesn't really matter. But I think there's larger improvements to be made to clean up the whole thing and keep them all in line.